holidays/countries/mongolia.py

```
@@ -0,0 +1,80 @@
+        # Renamed on December 23rd, 2011.
+        if self._year >= 2007:
+            self._add_holiday_dec_29(
+                # Restoration of Freedom and Independence Day.
+                tr("Үндэсний эрх чөлөө, тусгаар тогтнолоо сэргээсний баярын өдөр")
+                if self._year >= 2011
+                # Independence Day.
+                else tr("Үндэсний эрх чөлөөний өдөр")
+            )
+
+
+class MN(Mongolia):
```

Pylint (3.3.7)

[convention] Missing class docstring

(C0115)